### PR TITLE
VB Template fixes

### PR DESCRIPTION
**Pull request by mrlacey**
For #1539 (remove use of InlineAssignHelper)
Also includes some other minor bug fixes for the VB templates
- Fix for Icons on MasterDetail page not showing
- Fix for crash when navigating to ImageGallery in MVVMLight projects
- Standardized formatting of property definitions
- Added missing adaptive trigger to ImageGallery
- Added missing adaptive trigger to Grid page (C# AND VB)

### CODE REVIEW

**Comment by sibille** [comment]
@mrlacey, I don't know how that happened, but your last commit undo's a lot of changes that I made this morning in PR https://github.com/Microsoft/WindowsTemplateStudio/pull/1554.

Could you update your local branch again with dev to un-undo them?

File: templates/Features/DragAndDrop/.template.config/cs-CZ.template.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
   "author": "Microsoft",
   "name": "Drag & Drop",
-  "description": "The Drag & Drop feature provides a service to simplify the creation of drag and drop ready apps.",
+  "description": "The Drag & Drop feature provides a wrapper service over the standard UWP Drag and Drop functionallity to simplify the required code to create drag and drop ready apps.",
```
Comment (sibille) [inline templates/Features/DragAndDrop/.template.config/cs-CZ.template.json]: Description was shortened to make it fit in the box

File: templates/Features/MultiView/Services/ViewLifetimeControl.cs [hunk 2]
```diff
@@ -44,7 +43,7 @@ public sealed class ViewLifetimeControl
 
                 if (releasedCopy)
                 {
-                    throw new InvalidOperationException("ExceptionViewLifeTimeControlViewDisposal".GetLocalized());
```
Comment (sibille) [inline templates/Features/MultiView/Services/ViewLifetimeControl.cs]: Added exception messages to string resources

File: templates/_catalog/cs-CZ.frameworks.json [hunk 3]
```diff
@@ -18,10 +18,5 @@
     "name": "CaliburnMicro",
     "displayName": "Caliburn.Micro",
     "summary": "Caliburn.Micro je oblíbený rámec (framework) MVVM, který klade důraz na konvence a uspořadatelnost."
-  },
```
Comment (sibille) [inline templates/_catalog/cs-CZ.frameworks.json]: Added Prism framework

**Comment by mrlacey** [comment]
@sibille I think this update should address the issue.

**Comment by sibille** [comment]
@mrlacey, I still see my changes been undone with this PR.
image 
For the drag and drop feature the first description is the correct one for example. 
I think the problem is that your dev branch that is out of sync with the WTS dev branch. Did you fetch the upstream changes?

**Comment by mrlacey** [comment]
Originally I had merged in an old version of the dev branch but tried to go back to an earlier commit and then merge from Microsoft/dev but something obviously didn't work. The rush of commits merged to the main branch over the last few days have made managing merge conflicts unworkable. 
Instead going to cancel this and start again.

**Comment by ralarcon** [comment]
Ok @mrlacey, I hope you don't need to make too much re-work.

**Pull request closed without merging**